Trappist/src/Promact.Trappist.DomainModel/ApplicationClass/SeedDatabase.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Promact.Trappist.DomainModel.ApplicationClass;
+using Promact.Trappist.DomainModel.Models.Question;
+using Promact.Trappist.Web.Data;
+using System;
+using System.Linq;
+
+namespace Promact.Trappist.DomainModel.Data
+{
+
+    /// <summary>
+    /// Polpulates pre-required table
+    /// </summary>
+    public class SeedDatabase
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1118)